apps/testclient/templates/home.html

```diff
@@ -41,11 +46,33 @@ <h2>Step 1: Sample Authorization</h2>
 			<br />
 
 			<a href="{% url 'authorize_link' %}" class="ds-c-button ds-u-margin-y--2 ds-c-button--primary ds-u-color--white">Get a Sample Authorization Token</a>
+			{% switch "bfd_v2" %}
```

Comment:
As discussed, we want to prevent the general public from using V2 in the testclient at this time so we should replace this switch with another switch like `bfd_v2_testclient`

Reply:
a testclient specific switch can be added to achieve this.